src/apps/frontend/pages/tasks/task-list.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Task } from '../../types/task';
+import MenuItem from '../../components/menu-item';
+import { Button } from '../../components';
+import { ButtonKind } from '../../types/button';
+
+const TaskList: React.FC<{ tasks: Task[] }> = ({ tasks }) => (
+  <div className="p-4 mx-auto md:p-6 border border-[#e4e7ec]">
+    <h2 className="text-base font-medium text-black">To Do</h2>
```

Comment:
ideally we should have H2 component so font weight and color need not be specified here.

Open to use any library as reference, this provides a [good abstraction](https://baseweb.design/components/typography/)